app/containers/markdown/index.js

```diff
@@ -151,7 +151,7 @@ export default class Markdown extends PureComponent {
 				{literal}
 			</Text>
 		);
-	}
+	};
```

Comment:
Can you remove these semicolons?

Reply:
yes, it was my prettier extension, sorry xD